services/movie.py

```diff
@@ -0,0 +1,45 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: int = None,
+        actors_ids: int = None
+) -> QuerySet[Movie]:
+    movies = Movie.objects.all()
+
+    if genres_ids and actors_ids:
+        movies = movies.filter(
+            genres__id__in=genres_ids,
+            actors__id__in=actors_ids
+        )
+    elif genres_ids:
+        movies = movies.filter(genres__id__in=genres_ids)
+    elif actors_ids:
+        movies = movies.filter(actors__id__in=actors_ids)

```

Comment:
Use just 2 conditions,
1) when genres_ids is not empty
2) when actors_ids is not empty